beta/examples/tensorflow/segmentation/train.py

```diff
@@ -46,6 +47,11 @@ def get_argument_parser():
                                         print_freq=False,
                                         dataset_type=False,
                                         cpu_only=False)
+    parser.add_argument('--mode', '-m',
```

Comment:
Seems, this is a workaround for the sanity test.

Reply:
Yes, I fixed it.